tests/sc_validate_table_name.test/t0_long_name.req

```diff
@@ -0,0 +1 @@
+create table t2345678901234567890123456789012 (i int)
```

Comment:
I like the names of your test files! I often give them very undescriptive names like t00.req